src/Components/Web/test/Forms/InputRenderer.cs

```diff
@@ -0,0 +1,25 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components.RenderTree;
+using Microsoft.AspNetCore.Components.Test.Helpers;
+
+namespace Microsoft.AspNetCore.Components.Forms
+{
+    internal static class InputRenderer
+    {
+        public static async Task<TComponent> RenderAndGetComponent<TValue, TComponent>(TestInputHostComponent<TValue, TComponent> hostComponent) where TComponent : InputBase<TValue>
```

Comment:
```suggestion
        public static async Task<TComponent> RenderAndGetComponent<TValue, TComponent>(TestInputHostComponent<TValue, TComponent> hostComponent) 
        where TComponent : InputBase<TValue>
```

Just to avoid GitHub's horizontal scrollbar